### PR TITLE
update architecture diagram

### LINKED ISSUE #379: Update architecture doc
Priority: Low

## Issue Description

The [architecture doc](https://github.com/Kuadrant/kcp-glbc/blob/main/docs/architecture.svg) in the kcp-glbc repo hasn't been updated in a while, and a few changes have happened recently that affect it, in particular the change to move everything into a single workspace

### PR DESCRIPTION
**Pull request by maleck13**
Closes #379

### CODE REVIEW

**Comment by maleck13** [comment]
cc @mikenairn @david-martin

File: docs/architecture.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 The following diagram illustrates the high level architecture:
 
-![Deployment via KCP](architecture.svg)
+![Deployment via KCP](architecture-local-setup.png)
```
Comment (david-martin) [inline docs/architecture.md:5]: Do we want to keep the older one in the repo as well?

Comment (maleck13) [reply docs/architecture.md:5]: good spot removed

**Review by mikenairn — APPROVED** [review]
Quite hard to read without zooming in, but the contents of it when you do look good.

**Pull request merged**